lib/arena.rb

```diff
@@ -1,2 +1,40 @@
 class Arena
+  attr_accessor :name, :gladiators
+
+  def initialize (name)
+
+    @name = name.capitalize
+    @gladiators = []
+
+
+  end
+
+  def add_gladiator (name)
+    if @gladiators.length < 2
+     @gladiators << name
+    else
+      "You have 2 Gladiators!"
+    end
+  end
+
+  def fight
+    if gladiator =0 then action in arena = nil
+    if @gladiators.count = 2
+      return nil
+    end
+
+    if spear vs trident return = "spears win"
```

Comment:
Where do we get the spear or trident from?  Don't we need to ask a gladiator for its weapon?
``` ruby
gladiator = gladiators.first
gladiator.weapon
```